blocks/init/src/Blocks/components/button/manifest.json

```diff
@@ -1,4 +1,5 @@
 {
+	"$schema": "./../../../../node_modules/@eightshift/frontend-libs/schemas/component.json",
```

Comment:
I think you should be able to just point to the full URL of the `component.json` file like:

```json
"$schema": "https://github.com/infinum/eightshift-frontend-libs/blob/main/schemas/variation.json",
```

That's how it's referenced in .org's `theme.json`

<img width="478" alt="image" src="https://user-images.githubusercontent.com/8638515/150758585-ada6111e-edcc-4e3e-a395-c0e64166e633.png">

I've set the branch name to main, but it can be develop (although it should be main, as that is stable).

Reply:
I tried. You must point to actual json file so raw. but there is some caching issue here and it is not working correctly

Reply:
It's probably not working locally because you haven't committed it yet 🤷🏼‍♂️

Reply:
I did but still not working. :(